rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -76,25 +76,27 @@ void RSocketStateMachine::setResumable(bool resumable) {
   remoteResumeable_ = isResumable_ = resumable;
 }
 
-bool RSocketStateMachine::connectServer(
+void RSocketStateMachine::connectServer(
     yarpl::Reference<FrameTransport> frameTransport,
     const SetupParameters& setupParams) {
   setResumable(setupParams.resumable);
-  return connect(std::move(frameTransport), true, setupParams.protocolVersion);
+  CHECK(connect(std::move(frameTransport), setupParams.protocolVersion));
+  sendingPendingFrames();
 }
 
+//TODO: ensure the return value is not ignored!
```

Comment:
What's the point of this TODO?  How are we going to ensure the return value isn't ignored here?  With C++17's `[[nodiscard]]`?